packages/server/test/unit/project_spec.js

```diff
@@ -698,6 +698,17 @@ This option will not have an effect in Some-other-name. Tests that rely on web s
       })
     })
 
+    // ? isn't tested here because it's valid in Unix/Linux, but invalid in Windows
```

Comment:
Could we have a test that only runs in Unix/Linux, using a conditional with `process.platform`? If the presence of the file itself is a problem, we could dynamically create it only in that case too instead of committing a file with a `?` to the repo.

Reply:
I've never thought about that. I'll do it.

Reply:
It seems that `?` is only restricted on Windows. So, I've made the condition, `process.platform !== 'win32'`.